app/models/custom-form.js

```diff
@@ -51,7 +51,7 @@ export default ModelBase.extend({
   fieldIdentifier : attr('string'),
   form            : attr('string'),
   type            : attr('string', { defaultValue: 'text' }),
-  name            : attr('string'),
+  _name           : attr('string'),
```

Comment:
This won't work

Reply:
i have defined a computed property `name` which will use `_name`

Reply:
Won't work. Try the custom forms and attach a GIF

Reply:
are you referring to the forms  under session & speaker details?

Reply:
Yes